tests/run_examples.py

```diff
@@ -203,27 +196,31 @@ def get_exe_cmd(ex, nodist, nostream):
             return None
 
     if os.path.dirname(ex).endswith("daal4py"):
```

Comment:
i think we also have to have if for "mb" directory 
```suggestion
    if os.path.dirname(ex).endswith("daal4py") or os.path.dirname(ex).endswith("mb"):
```

Or might be add as separate if line

Reply:
Good call!